vmdk_plugin/utils/log_formatter/log_formatter.go

```diff
@@ -17,12 +17,15 @@
 // * storage team specifications. The `appendKeyValue` and `needsQuoting` functions are copied
 // * from the implementation of the TextFormatter in Logrus.
 
-package main
+package log_formatter
 
 import (
 	"bytes"
 	"fmt"
 	log "github.com/Sirupsen/logrus"
+	"github.com/natefinch/lumberjack"
+	"github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/config"
```

Comment:
I guess this is the style that is used by vDVS so far :)
But to double-check, what do you mean by keeping vendor imports separate?